workspaces/optic-engine/src/queries/snapshots/optic_engine__queries__shape__test__can_generate_remove_field_commands__choice_mapping.snap

```diff
@@ -8,7 +8,14 @@ expression: choice_mapping
             ObjectChoice {
                 shape_id: "object_shape_1",
                 json_type: Object,
-                fields: [],
+                fields: [
+                    ObjectFieldChoice {
+                        name: "lastName",
+                        field_id: "field_1",
+                        shape_id: "string_shape_1",
+                        is_removed: true,
+                    },
+                ],
```

Comment:
This is the most important snapshot to review - the others just add `is_removed` to the all the keys. We expect removed nodes to be displayed in the spectacle shapeViewerProjection, however they have the key `is_removed` which can then be appropriately removed from the FE (similar to how endpoints, requests and responses are handled)